docs/BreakingChanges/071 SqlBulkCopy uses destination column encoding for strings.md

```diff
@@ -0,0 +1,23 @@
+## 71: SqlBulkCopy uses destination column encoding for strings
```

Comment:
{
  "Id": "71",
  "Title": "SqlBulkCopy uses destination column encoding for strings",
  "Details": "When inserting data into a column, SqlBulkCopy uses the encoding of the destination column rather than the default encoding for `VARCHAR` and `CHAR` types. This change eliminates the possibility of data corruption caused by using the default encoding when the destination column does not use the default encoding. In rare cases, an existing application may throw a SqlException exception if the change in encoding produces data that is too big to fit into the destination column.",
  "Suggestion": "Expect that SqlBulkCopy will no longer corrupt data due to encoding differences. If strings near the destination column's size limit are being copied, it may be necessary to either pre-encode data (to be copied to check that the data will fit in the destination column) or catch SqlExceptions.",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887(v=vs.110).aspx?f=255&MSPPError=-2147217396#xml",
  "Markdown": null,
  "ApplicableApis": [
    "T:System.Data.SqlClient.SqlBulkCopy",
    "M:System.Data.SqlClient.SqlBulkCopy.#ctor(System.Data.SqlClient.SqlConnection)"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [x] T:System.Data.SqlClient.SqlBulkCopy
- [x] M:System.Data.SqlClient.SqlBulkCopy.#ctor(System.Data.SqlClient.SqlConnection)